source/_integrations/cast.markdown

```diff
@@ -109,7 +109,7 @@ media_player:
   type: list
   keys:
     uuid:
-      description: UUID of a Cast device to add to Home Assistant. Use only if you don't want to add all available devices. The device won't be added until discovered through mDNS.
+      description: UUID of a Cast device to add to Home Assistant. Use only if you don't want to add all available devices. The device won't be added until discovered through mDNS. In order to find the UUID for your device install pyChromecast on a computer or enter the Home Assistant Docker container and type the following changing the friendly names as required for your situation. python3 -c "import pychromecast; print(pychromecast.get_listed_chromecasts(friendly_names=["Living Room TV", "Bedroom TV", "Office Chromecast"]))"
```

Comment:
So how do users of the OS installation method do this? Or Supervised? And users running the container in Docker?

Why not simply use an mDNS browser?

![image](https://user-images.githubusercontent.com/195327/95177106-15737180-07be-11eb-9326-c4cb6b4f3af8.png)

Reply:
What I did, and what I tried to say in brief in my suggested change, is I entered the HA docker container and ran the pychromecast command there. In all the different installation options the user just has to get to where pychromecast is installed, I appreciate that is not always easy.

So that is one option. Your option is also good so perhaps we mention both of these options? Gives people choice as to what resources they have available to get the UUID?

Reply:
The only thing I notice is with your browser is that the UUID is not broken up into the segments (separates by hyphens) that mine were. I don’t know if that makes a difference when you put it into the config?

Reply:
Just following up. Did you want me to add the mDNS browser option?

Reply:
I think we should find options that make it accessible for users of any level. Executing Python like that isn't that friendly.

So any other option should be preferred IMHO.

Reply:
OK I have changed my suggestion to first direct users to the mDNS browser option but kept a quick reference to using pychromecast. Thoughts?